tcases-core/src/main/java/org/cornutum/tcases/io/SystemTestHtmlWriter.java

```diff
@@ -311,21 +307,19 @@ protected void writeDefaultStyle()
     xmlWriter_.writeElementStart( "STYLE");
     xmlWriter_.indent();
 
-    LineIterator styleLines = null;
     try
       {
-      for( styleLines = IOUtils.lineIterator( getClass().getResourceAsStream( "system-test.css"), "UTF-8");
-           styleLines.hasNext();
-           xmlWriter_.println( styleLines.next()));
+      BufferedReader in = new BufferedReader( new InputStreamReader( getClass().getResourceAsStream( "system-test.css"), "UTF-8"));
```

Comment:
Oops, classic defect here: does not ensure input stream is always closed, even after an exception. For example, the original `finally` clause is missing. 

But instead of `IOUtils.closeQuietly`, should now use standard Java try-with-resources.

Reply:
fixed